catroid/src/main/java/org/catrobat/catroid/stage/StageListener.java

```diff
@@ -251,7 +251,7 @@ public void create() {
 
 	public void cloneSpriteAndAddToStage(Sprite cloneMe) {
 		Sprite copy = cloneMe.cloneForCloneBrick();
-		copy.resetSprite();
+		//copy.resetSprite(); // can I safely remove this?
```

Comment:
No, your guess was quite good. At least with certain bricks a ClassCastException was thrown (while trying to cast a `Look` (base-type) as a `PhysicsLook`).